modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1267,6 +1269,30 @@ private boolean doComputeContains(double localX, double localY) {
         return TransformedShape.translatedShape(shape, x, y);
     }
 
+    /**
+     * The size of a tab stop in spaces.
```

Comment:
"tab stop" seems to be an uncommon term. Better are "horizontal tab" (used in the [JLS](https://docs.oracle.com/javase/specs/jls/se13/html/jls-3.html#jls-3.10.6)), "tab character" (used in [`Pattern`](https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/util/regex/Pattern.html), "horizontal tabulation" or the like.

Reply:
The terms "tab character" or "horizontal tab" refer to the ASCII tab character itself. Since a tab character isn't a fixed number of spaces, changing it to "size of a tab character" could be misleading. I'd be fine with another alternative, though, if someone could come up with a better one.

Reply:
We are referring to the character here aren't we ? ie the actual character and the rest of it is about how it renders.
Paraphrasing the java doc it says :
If you display <the tab character> it will display as N instances of <the space character>

Reply:
I find "size of the tab character" to be a bit confusing.  The number of space you advance is not a constant for one, so tab characters don't really have a fixed size in that sense.  The javadoc could be rephrased as "the distance between tab stops".  However, that keeps the unfamiliar, but correct IMO, "tab stop" wording.  For the record the top google hit for "tab stop" results in: "A tab stop is a term used to describe the location the cursor stops after the Tab key is pressed." Wikipedia says something similar and adds, "In text editors on a computer, the same concept is implemented simplistically with automatic, fixed tab stops."

Reply:
OK. Lets not do this to death. I knew what you meant by tab stop.